packages/dev/core/src/Meshes/Compression/dracoEncoder.ts

```diff
@@ -0,0 +1,292 @@
+import { _IsConfigurationAvailable, DracoCodec, type IDracoCodecConfiguration } from "./dracoCodec";
+import { DracoAttributeName, DracoEncoderMethod } from "./dracoEncoder.types";
+import type { EncoderMessage, IDracoAttributeData, IDracoEncodedMeshData, IDracoEncoderOptions } from "./dracoEncoder.types";
+import { EncodeMesh, EncoderWorkerFunction } from "./dracoCompressionWorker";
+import { Tools } from "../../Misc/tools";
+import { VertexBuffer } from "../buffer";
+import type { Nullable } from "../../types";
+import { Mesh } from "../mesh";
+import type { Geometry } from "../geometry";
+import { Logger } from "../../Misc/logger";
+import { deepMerge } from "../../Misc/deepMerger";
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+declare let DracoEncoderModule: any;

```

Comment:
Is there any reason this has to be `any`? Can we not define a type for this? Or if it an opaque value (doesn't seem to be the case from its usage in this code though), then it could be `unknown`?

Reply:
I touched on this in a different file that has the same declaration (maybe should add a comment for it here too), but the @types/draco3d package includes a type for DracoDecoderModule but not DracoEncoderModule, despite it seeming to exist in the real module once imported. Not sure why.

Let me see about changing this to unknown :)

Reply:
Actually, we might be able to define our own type for it... I'll look :)

Reply:
If they don't have types, we should still be able to define the types ourselves at least.